content/run.md

```diff
@@ -162,9 +167,12 @@ The compset chosen here is I2000Clm50BgcCropGs. Initialize the model for the yea
     
 ### Inputdata
 
-Add domain file path following below commands in the case directory:
+Change data atmosphere domain (ATM_DOMAIN_PATH) and land domain (LND_DOMAIN_PATH) file paths following below commands in the case directory:
 
```

Comment:
I still think we should explain all environmental path variables that we use properly. 

Could you also describe what the variables 

    $ATMDOM, 
    $MYREGDATA 

does?